xdsl/dialects/bufferization.py

```diff
@@ -103,6 +140,51 @@ def __init__(
             result_types=(result_type,),
         )
 
+    @classmethod
```

Comment:
Does this not work?
```
assembly_format = "`(` $dynamic_sizes `)` ( `copy` `(` $copy^ `)`)? (`size_hint` `=` $size_hint^)? attr-dict `:` type($result)
```

Reply:
It won't print exactly what MLIR prints but I think that's ok, the difference is printing whitespace around the `=`